### DISCUSSION TITLE
the account is   owner . it will go back the page after click project

### DISCUSSION THREAD

**Discussion by z7210073** [nocodb/nocodb#320] [Q&A]
I run it in kubernetes,mount pvc /usr/src/app/noco.db and /usr/src/app/nc.
I use the account that is owner  click its project but it will go back the page

**Comment by o1lab** [comment]
I doubt if its rightly mounted or it doesn't has the write permissions to sqlite. So our handshake is thinking a SQLite doesn;t existi and going to start page.

Instead of mounting for SQlite, you can use NC_DB URL and set it to actual database schema itself.
